build-logic/src/main/kotlin/io.sentry.spotless.gradle.kts

```diff
@@ -0,0 +1,37 @@
+import com.diffplug.spotless.LineEnding
+
+plugins {
+    id("com.diffplug.spotless")
+}
+
+spotless {
+    lineEndings = LineEnding.UNIX
+    java {
+        target("**/*.java")
+        removeUnusedImports()
+        googleJavaFormat()
+        targetExclude("src/**/java/io/sentry/vendor/**")
+    }
+    kotlin {
+        target("**/*.kt")
+        ktlint()
+        targetExclude("src/test/java/io/sentry/apollo4/generated/**", "src/test/java/io/sentry/apollo3/adapter/**")
+        suppressLintsFor {
+            step = "ktlint"
+            shortCode = "standard:max-line-length"
```

Comment:
`max-line-length`, `property-naming` and `function-naming` had hundreds of violations that aren’t auto-fixable and would have made this PR even larger.